drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/AbstractKogitoDMNService.java

```diff
@@ -90,10 +89,21 @@ protected ClientDMNType getDMNTypeFromMaps(final Map<String, ClientDMNType> dmnT
         return dmnTypesMap.get(typeRef);
     }
 
+    /**
+     * This method retrieves all DMN Types are composed by
+     * - Primitive FEEL types, listed inside <code>BuiltInType</code> enum
+     * - User defined Types, which are available inside inside <code>jsitItemDefinitions</code> list
+     * @param jsitItemDefinitions
```

Comment:
Shouldn't it be jsItItemDefinition?

Reply:
@dupliaka `jsitItemDefinitions` is the list name, I think it's correct.